browser/ui/views/download/brave_download_item_view.cc

```diff
@@ -150,8 +150,8 @@ std::u16string BraveDownloadItemView::CalculateAccessibleName() const {
   if (!origin_url_text_.empty()) {
     std::u16string extra;
     if (!is_origin_url_secure_) {
-      extra += char16_t(' ') + brave_l10n::GetLocalizedResourceUTF16String(
-                                   IDS_NOT_SECURE_VERBOSE_STATE);
+      extra += char16_t(' ') +
+               l10n_util::GetStringUTF16(IDS_NOT_SECURE_VERBOSE_STATE);
     }
```

Comment:
(again not for this PR) I guess this is probably okay as this is extra detail

Reply:
https://github.com/brave/brave-browser/issues/45771